src-java/testing/test-library/src/main/java/org/openkilda/testing/service/traffexam/model/EndpointReport.java

```diff
@@ -38,7 +37,7 @@ public EndpointReport(String error) {
     }
 
     public EndpointReport(ReportResponse report) {
-        if (Strings.isEmpty(report.getError())) {
+        if (report.getError() == null || report.getError().isEmpty()) {
```

Comment:
We could also use `org.apache.commons.lang3.StringUtils.isEmpty()` check here, since we already have implementation("org.apache.commons:commons-lang3") as a dependency